### PR TITLE
fix travis builds and Symfony deprecation

### PR DESCRIPTION
**Pull request by craigh**

### CODE REVIEW

**Comment by craigh** [comment]
@Kocal @blankse @monteiro 

can someone merge this (and then #294)? I would be great to get this bundle functional...

cc @stof

**Comment by craigh** [comment]
@monteiro I see you 👍 my comment, but you did not merge. Can you do that?

File: .travis.yml [hunk 1]
```diff
@@ -27,8 +27,6 @@ script:
 matrix:
     fast_finish: true
     include:
-      - php: 7.2
-        env: SYMFONY_VERSION='^3.4.0'
```
Comment (pierredup) [inline .travis.yml:31]: This bundle still supports Symfony 3.4, why was this removed?

Comment (craigh) [reply .travis.yml:31]: I guess because I misread it. If it can be re-added and the builds/tests pass, then great
